js/viz/translators/translator2d.js

```diff
@@ -149,9 +150,19 @@ function isEqualRange(range) {
     const businessRange = that.getBusinessRange();
     const canvasOptions = getCanvasBounds(businessRange);
     const correctionPrecision = getEqualityCorrection(businessRange) / 100;
+    const comparingRange = range && new rangeModule.Range(range);
 
-    return range && _abs(range.min - canvasOptions.rangeMin) <= correctionPrecision &&
-        _abs(range.max - canvasOptions.rangeMax) <= correctionPrecision;
+    if(businessRange.axisType === "discrete") {
+        return false;
+    }
+
+    if(comparingRange && businessRange.axisType === "logarithmic") {
+        comparingRange.min = comparingRange.min && vizUtils.getLog(comparingRange.min, businessRange.base);
```

Comment:
You can use defined 'getLog' variable instead vizUtils.getLog.

Reply:
done